client/package.json

```diff
@@ -6,7 +6,7 @@
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
-    "bootstrap": "^5.0.1",
```

Comment:
Not quite sure which bootstrap to use. Looks like the original one we had is a later version.